src/style.css

```diff
@@ -1,3 +1,77 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+  align-self: center;
+}
+
+.logo img {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  width: 100%;
+  height: 60px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding: 23px 0;